PocketKit/Sources/PocketKit/MyList/SearchItemsList/SearchView.swift

```diff
@@ -59,7 +61,7 @@ struct RecentSearchView: View {
                     }
                     .contentShape(Rectangle())
                     .onTapGesture {
-                        // TODO: Handle recent search tap
+                        viewModel.updateSearchFromRecentSearch(with: recentSearch)
```

Comment:
what do you think about doing `viewModel.searchText = recentSearch` here instead of creating a function?

Reply:
we'd still need to call the search function, so im not sure if we're solving anything by adding the change you suggested.

Reply:
nvm, i see the changes you requested for `SearchViewModel`